oracle/README.md

```diff
@@ -242,9 +197,9 @@ instances:
 
 2. Update the `sqlnet.ora`, `listener.ora`, and `tnsnames.ora` to allow TCPS connections on your Oracle Database. 
 
-##### TCPS through the Oracle Instant Client
+##### TCPS through the Oracle
```

Comment:
```suggestion
##### TCPS through Oracle
```

Reply:
Offering this as a suggestion, but please update if something else would be more apt here.